.dockerignore

```diff
@@ -1,9 +1,10 @@
 /t
 /testdata
-.concourse.yml
-.gitignore
-.bot.log
-LICENSE
-Makefile
-prometheus_bot
-README.md
\ No newline at end of file
+/.concourse.yml
```

Comment:
What's the reason for changing these and adding a `/` at the beginning of them?

Reply:
Because these files located only in root directory of repo.
Explicit more reliable and expected.